README.md

```diff
@@ -1103,14 +1103,20 @@ column type.
 #### Convert a column from text to json
 
 Postgres' `alter column` command will not automatically convert a `text`
-column to `json`, but it can still be done with plain SQL.
+column to `json`, but it can still be done with a piece of plain SQL.
```

Comment:
If we're going to provide an AR migration here instead of plain SQL, I think we can delete this paragraph.